app-emulation/fex-xtajit/fex-xtajit-2506.ebuild

```diff
@@ -35,9 +35,10 @@ BDEPEND="
 	llvm-core/llvm
 "
 
-PATCHES="
-	${FILESDIR}/${P}-constness.patch
-"
+src_prepare() {
+	has_version '<dev-util/llvm-mingw64-13.0.0' && eapply "${FILESDIR}/${P}-constness.patch"
```

Comment:
Conditional patches really should be avoided wherever possible: https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#conditional-patching

Reply:
Yea, it is not great, but it is build time only anyway.

Reply:
OK, fine, with reluctance (but let's avoid it in future please unless we really must)